packages/core/src/lint.ts

```diff
@@ -89,3 +91,46 @@ export async function lintDocument(opts: {
   });
   return ctx.problems.map((problem) => config.addProblemToIgnore(problem));
 }
+
+export async function LintDocumentConfig(opts: {
```

Comment:
should start from lowercase, also, rename to simply `lintConfig`

Reply:
I should be more accurate. Thanks